### PR TITLE
🚒🏎️ Reordered connection sequence preventing race condition

### LINKED ISSUE #97: Unable to clone GitLab repos
I seem to be hitting a race in `GitlabService`'s initialization:
```
$ git repo -v lab clone guyzmo/git-repo
Traceback (most recent call last):
  File "…/git_repo/repo.py", line 347, in do_clone
    service = RepositoryService.get_service(repository, self.target)
  File "…/git_repo/services/service.py", line 126, in get_service
    cls._current = service(repository, config)
  File "…/git_repo/services/ext/gitlab.py", line 22, in __init__
    super().__init__(*args, **kwarg)
  File "…/git_repo/services/service.py", line 168, in __init__
    self.connect()
  File "…/git_repo/services/ext/gitlab.py", line 26, in connect
    self.gl.set_url(self.url_ro)
AttributeError: 'GitlabService' object has no attribute 'gl'
```

A brief look at the code suggests the problem lies in the execution order:
- `GitlabService.__init__` calls `RepositoryService.__init__`
  - `RepositoryService.__init__` initializes `RepositoryService.insecure`
  - `RepositoryService.__init__` calls `GitlabService.connect`
    - `GitlabService.connect` accesses `GitlabService.gl`
- `GitlabService.__init__` initializes `GitlabService.gl` using `RepositoryService.insecure`

The other services swap the two lines in the constructor, but here we can't do that.

I'm using git-repo 1.8.0 on Python 3.6, installed via `pip install --user git-repo`. `list` and `create` both work fine.

### PR DESCRIPTION
**Pull request by guyzmo**
fixes #97

### CODE REVIEW

File: git_repo/services/ext/gitlab.py [hunk 1]
```diff
@@ -19,10 +19,11 @@ class GitlabService(RepositoryService):
     fqdn = 'gitlab.com'
 
     def __init__(self, *args, **kwarg):
+        self.gl = gitlab.Gitlab(self.url_ro)
         super().__init__(*args, **kwarg)
-        self.gl = gitlab.Gitlab(self.url_ro, ssl_verify=not self.insecure)
 
     def connect(self):
+        self.gl.ssl_verify = self.insecure
```
Comment (AluisioASG) [inline git_repo/services/ext/gitlab.py]: `= not self.insecure`

Comment (guyzmo) [inline git_repo/services/ext/gitlab.py]: oh my! thank you 😖 /o\ 😳🕳️

Comment (guyzmo) [inline git_repo/services/ext/gitlab.py]: fixing it tomorrow first thing, it's quite late to do any coding ;)

**Comment by guyzmo** [comment]
done and merged in master. I'll publish a 1.8.1 ASAP with that fix.

**Pull request closed without merging**